lms/static/js/jquery.gradebook.js

```diff
@@ -27,7 +27,10 @@ var Gradebook = function($element) {
 
 	var moveDrag = function(e) {
 		var offset = e.pageX - mouseOrigin;
-		var targetLeft = clamp(tableOrigin + offset, -maxScroll, 0);
+		// Updating direction of scroll by calculating pixels. O me left most and
```

Comment:
I still don't understand what you mean by "updating direction of scroll". By "scroll", do you mean moving the gradebook across the page? Normally, I think of "scroll" as the user changing the viewport, so they can view a different part of the page. It doesn't involve elements on the page moving.

Also, what do you mean by "updating direction"? Are we in the middle of a scroll, and we suddenly need to change direction?

Also, you have a typo in this line: I think you mean "0 means left-most" instead of "0 me left most".